server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -111,14 +111,70 @@ protected boolean checkHasMandatoryOptions(List<?> rules, ParserRuleContext ctx,
     return true;
   }
 
+  protected void checkHasMandatoryOptions(TerminalNode rules, ParserRuleContext ctx, String options) {
+    ArrayList<TerminalNode> tempList = new ArrayList<>();
+    tempList.add(rules);
+    checkHasMandatoryOptions(tempList, ctx, options);
```

Comment:
This looks equivalent to just 
```
      throwException(
          ErrorSeverity.ERROR,
          VisitorUtility.constructLocality(ctx, context),
          "Missing required option: ",
          options);
```

Reply:
Replaced the one usage of this in the latest commit.